packages/api/src/services/lsp-state.e2e.ts

```diff
@@ -0,0 +1,81 @@
+require("dotenv").config();
```

Comment:
this file is an end to end test which tests the lsp state service, just to make sure we get data back we expect. uses mainnet deployed contract.